docs/variants/protectli_vp2420/releases.md

```diff
@@ -0,0 +1,52 @@
+# Release Notes
+
+Following Release Notes describe status of open-source firmware development for
+Protectli VP2420 family.
+
+For details about our release process please read
+[Dasharo Standard Release Process](../../dev-proc/standard-release-process.md).
+
+<center>
+[Subscribe to Protectli VP2420 Dasharo Release Newsletter]
+[newsletter]{.md-button .md-button--primary .center}
+</center>
+
+Test results for this platform can be found
+[here](https://docs.google.com/spreadsheets/d/1wI0qBSLdaluayYsm_lIa9iJ9LnPnCOZ9eNOyrKSc-j4/edit#gid=975611333).
+
+## v1.0.0 - 2022-12-22
+
+### Added
+
+- Added support for vp2420
+
+### Binaries
+
+[protectli_VP2420_v1.0.0.rom][v1.0.0_rom]{.md-button}
+[sha256][v1.0.0_hash]{.md-button}
+[sha256.sig][v1.0.0_sig]{.md-button}
+
+How to verify signatures:
+
+```bash
```

Comment:
@Stojek139808 3 first links do not work. It happens because the firmware name is: `protectli_vp2420_v1.0.0` not `protectli_vault_ehl_v1.0.0`

Reply:
`protectli_vp2420_v1.0.0` is the correct one

Reply:
Seems to work now, changed another the 3 newsletter links below too

Reply:
Good, thanks